packages/extension/src/components/molecules/NetworkIndicator/NetworkIndicator.tsx

```diff
@@ -70,7 +84,24 @@ const NetworkDisplay: FC<NetworkDisplayProps> = ({
               {description}
             </Typography>
           </Box>
-          {isSelected ? <CheckIcon /> : null}
+          <Box>
+            {onRemove && (
+              <div
```

Comment:
Instead of using a div, you can use IconButton:
https://mui.com/material-ui/react-button/#icon-button